app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -1,76 +1,82 @@
-    = hidden_field_tag :page, params[:page]
-    = hidden_field_tag :value_band_id, params[:value_band_id] if params[:value_band_id]
+  = hidden_field_tag :scheme, params[:scheme]
+  = hidden_field_tag :filter, params[:filter] if params[:filter]
+  = hidden_field_tag :page, params[:page]
+  = hidden_field_tag :value_band_id, params[:value_band_id] if params[:value_band_id]
 
-    %h2.heading-medium
-      = t('.allocate_claims')
+  %fieldset.inline
+    %legend
+      %span.heading-medium
+        = t('.allocate_claims')
     .grid-row.js-typeahead
       .column-one-third.js-typeahead
         = label_tag :quantity_to_allocate, t('.no_of_claims'), class: 'form-label-bold'
-        = number_field_tag :quantity_to_allocate, nil, { style: 'width:100%', min: 0, size: 5, class: 'form-control'}
+        = number_field_tag :quantity_to_allocate, nil, { style: 'width: 100%', min: 0, size: 5, class: 'form-control' }
       .column-one-third
         #cc-caseworker
-          = f.label :case_worker_id, t('dictionary.models.case_worker'), for: "allocation_case_worker_id_autocomplete", class: 'form-label-bold'
-          = f.collection_select :case_worker_id, @case_workers, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete' , style: 'width:100%'}
+          = label_tag :allocation_case_worker_id, t('dictionary.models.case_worker'), class: 'form-label-bold'
+          = collection_select :allocation, :case_worker_id, @case_workers, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', style: 'width: 100%' }
       .column-one-third
         %br/
-        = f.submit t('.allocate'), class: 'button allocation-submit'
+        = submit_tag t('.allocate'), class: 'button allocation-submit'
 
 
-    %h2.heading-medium
-      = t('.filter_claims')
+  %fieldset.inline
+    %legend
+      %span.heading-medium
+        = t('.filter_claims')
     .grid-row
       .column-one-third.ui-filter
         = render partial: 'filter_tasks'
       .column-one-third.ui-filter
         = render partial: 'filter_value_bands'
       .column-one-third
         %br/
-        %button.clear-filters.button.button-secondary{:type => 'button'}
+        %button.clear-filters.button.button-secondary{ type: 'button' }
           = t('.clear_filters')
 
-    %h2.heading-medium
-      = t('.allocation_queue')
-    %table#dtAllocation{summary: t('.table_summary')}
-      %thead
-        %tr
-          %th
-
-          %th
-            = t('.case_number')
-          %th
-            = t('.court')
-          %th
-            = t('.defendents')
-          %th
-            = t('.type')
-          %th
-            = t('.submitted')
-          %th
-            = t('.total')
+  %h2.heading-medium
+    = t('.allocation_queue')
+  %table#dtAllocation
+    %caption.visually-hidden
+      = t('.table_summary')
+      %noscript
+        = t('shared.not_available')
+    %thead
+      %tr
+        %th{ scope: 'col' }
+          = ''
```

Comment:
**question**: is this useful/necessary/semantic/accessible?

Reply:
In this instance, none of the above as our datatable plugin replaces this with within a 'select all' checkbox.

It however made the code more readable to me than just having the `%th`